NickvisionTubeConverter.Shared/Models/Download.cs

```diff
@@ -344,6 +343,7 @@ private void ProgressHook(Python.Runtime.PyDict entries)
         {
             using (Python.Runtime.Py.GIL())
             {
+                _pid = Python.Runtime.PythonEngine.GetPythonThreadID();
```

Comment:
Why moved this to progress hook?

Reply:
Once yt-dlp was running i was getting a different thread id when ffmpeg started. Implementing it like this will now also stop ffmpeg conversion whereas before ffmpeg was still running in the background.